sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidOperatorTable.java

```diff
@@ -35,20 +35,7 @@
 import org.apache.druid.java.util.common.StringUtils;
 import org.apache.druid.math.expr.BuiltInExprMacros;
 import org.apache.druid.sql.calcite.aggregation.SqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.ArrayConcatSqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.ArraySqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.AvgSqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.BitwiseSqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.BuiltinApproxCountDistinctSqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.EarliestLatestAnySqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.EarliestLatestBySqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.GroupingSqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.LiteralSqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.MaxSqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.MinSqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.StringSqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.SumSqlAggregator;
-import org.apache.druid.sql.calcite.aggregation.builtin.SumZeroSqlAggregator;
+import org.apache.druid.sql.calcite.aggregation.builtin.*;
```

Comment:
I'm a bit against including it as * and include only those libraries that we need, in future if we come up with another Agg that we do not use this wont be correct